services/api/routers/minio.py

```diff
@@ -43,6 +43,15 @@ def stream_minio_response(response):
         response.close()
         response.release_conn()
 
+def get_thumbnail_name(object_name: str, image_format: str):
+    """
+    :param object_name: the name of the file including path and extension
+    :param image_format: the mime-type of the image
```

Comment:
MIME type is usually a string like `'image/jpeg'`, i doubt that this is supposed to be appended to the object name - or is this really what you want? i think this is rather a validated extension derived from the mime-type (`file.content_type`) and looked up in `supported_image_formats`.